### PR TITLE
remove to_pe_constant option from CMT

### PR DESCRIPTION
**Pull request by ahiguera-mx**
Before you submit this PR: make sure to put all operations-related information in a wiki-note, a PR should be about code and is publicly accessible

**What is the problem / what does the code in this PR do**
The option `st.set_config(dict(gain_model=('CMT_model',("to_pe_constant",my_gains ))))` throws an error `InvalidDocument: cannot encode object: array()`, the reason is straxen does not support arrays as option for the metadata, therefore this option should be removed from CMT <br>
**Can you briefly describe how it works?**
see this example https://github.com/ahiguera-mx/xenon/blob/master/Remove_to_pe_constant_from_CMT.ipynb
**Can you give a minimal working example (or illustrate with a figure)?**
see this example https://github.com/ahiguera-mx/xenon/blob/master/Remove_to_pe_constant_from_CMT.ipynb

Please include the following if applicable:
  - Update the docstring(s)
  - Update the documentation
  - Tests to check the (new) code is working as desired.
  - Does it solve one of the open issues on github?

Please make sure that all automated tests have passed before asking for a review (you can save the PR as a draft otherwise).

### CODE REVIEW

File: straxen/get_corrections.py [hunk 1]
```diff
@@ -69,7 +67,7 @@ def get_to_pe(run_id, gain_model, n_tpc_pmts):
 
 FIXED_TO_PE = {
     # just some dummy placeholder for nT gains
-    'gain_placeholder': np.repeat(0.001, 494)
```
Comment (WenzDaniel) [inline straxen/get_corrections.py:72]: Just a question of taste, but I think 

```
np.ones(494)*0.0085
```

is a bit simpler.

**Review by WenzDaniel — COMMENTED** [review]
Looks fine

**Review by WenzDaniel — APPROVED** [review]

File: straxen/get_corrections.py (hunk 1 above)
Comment (jorana) [reply straxen/get_corrections.py:72]: My taste goes the other way ;) would only think about replacing the 494 to `straxen.n_tpc_pmts`. (I know I was the one who put it here in the first place, shame on me.)

**Review by jorana — APPROVED** [review]
Really nice Aaron, I think this makes it much cleaner! 

Just one last request, can you change the doc-string accordingly:
https://github.com/XENONnT/straxen/blob/5cff94ebc5ae75f65bcc50a6fd986f644cddcccf/straxen/corrections_services.py#L153

**Comment by jorana** [comment]
In the meanwhile, someone pointed out that without 5a01980 , we are getting some incomprehensible error when we are still trying to use keys that are not in FIXED_TO_PE (anymore). I propose to fix it like in 5a01980 such that people have a somewhat more transparent traceback.

File: straxen/get_corrections.py (hunk 1 above)
Comment (ahiguera-mx) [reply straxen/get_corrections.py:72]: done

**Comment by ahiguera-mx** [comment]
I've updated the docstrings accordingly and the last two suggestions `494 to straxen.n_tpc_pmts` and add the fix from [5a01980](https://github.com/XENONnT/straxen/commit/5a01980030917271aa01b35688214fd2cb481e84)

**Review by jorana — APPROVED** [review]
All looks good Aaron!

**Comment by jorana** [comment]
@ahiguera-mx @WenzDaniel I'm merging this

**Pull request merged**